src/gc/gcimpl.h

```diff
@@ -6,7 +6,7 @@
 #ifndef GCIMPL_H_
 #define GCIMPL_H_
 
-#define CLREvent CLREventStatic
+#define CLREvent GCEvent
```

Comment:
I saw a comment in `gcimpl.h` next to this define so I left it in place: `Please leave these definitions intact.` If it's not being used anywhere I'm happy to remove it but I'm not sure if it serves another purpose.